.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>spa</name>
```

Comment:
You can add these eclipse specific files to the list that is to be ignored. That way, we do not commit them.

Reply:
Yeah let's maybe add https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore to the gitignore? Some of these files are portable, some are not.

Reply:
👍 thanks for the link -- added that to the gitignore and removed the files